style/style.css

```diff
@@ -0,0 +1,681 @@
+.nav-banner {
+  display: none;
+}
+
+.navbar-nav .nav-link.active {
+  color: #f07f7e !important;
+}
+
+/* @media (min-width: 768px) {
```

Comment:
- [ ] Kindly remove all commented pieces of code in your documents to make your code cleaner.